darts/models/forecasting/rnn_model.py

```diff
@@ -86,6 +89,7 @@ def __init__(
         # The RNN module needs a linear layer V that transforms hidden states into outputs, individually
         self.V = nn.Linear(hidden_dim, target_size * nr_params)
 
+    @io_processor
```

Comment:
As discussed offline, RINorm might not be suitable for recurrent architectures (RNN & transformer, BlockRNN actually produce outputs by chunks). My guess is that it adds constraints on the hidden states which becomes unable to extract/store information efficiently.

If not enforcing `use_reversible_instance_norm=False`, I would maybe add a warning message when the model is instantiated?